feature-libs/cart/main/core/store/multi-cart-state.ts

```diff
@@ -18,5 +18,7 @@ export interface StateWithMultiCart {
 export interface MultiCartState {
   carts: StateUtils.EntityProcessesLoaderState<Cart | undefined>;
   active: string | null;
-  wishList: string;
+  index: {
+    [cartType: string]: string;
+  };
```

Comment:
What is the purpose of the index with a CartType of Active and Wishlist?  The `wishlist` and `active` first level fields seem to already fill this function.

Reply:
`wishlist` is moved to `index`. `active` still in first level, but will be moved in index in #14238